Homework_10/Task_2.py

```diff
@@ -0,0 +1,14 @@
+def is_prime(num):
+    if not 2 <= num <= 1000:
+        return False
```

Comment:
Can be better
    if number not in range(2, 1001):
        raise ValueError(f"{number} is not a valid number. Please, try any number in range from 2 to 1000.")